src/java/fr/paris/lutece/portal/service/mail/MailSenderDaemon.java

```diff
@@ -54,12 +56,16 @@
  */
 public class MailSenderDaemon extends Daemon
 {
+    protected static final String DAEMON_ID = "mailSender";
+
     private static final String PROPERTY_MAIL_HOST = "mail.server";
     private static final String PROPERTY_MAIL_PORT = "mail.server.port";
     private static final String PROPERTY_MAIL_DEAMON_WAITTIME = "mail.daemon.waittime";
     private static final String PROPERTY_MAIL_DEAMON_COUNT = "mail.daemon.count";
     private static final String PROPERTY_MAIL_USERNAME = "mail.username";
     private static final String PROPERTY_MAIL_PASSWORD = "mail.password";
+    private static final String PROPERTY_MAIL_DAEMON_RETRYONERROR_WAITTIME = "mail.daemon.retryonerror.waittime";
+    private static final String PROPERTY_MAIL_DAEMON_RETRYONERROR_WAITTIME_UNIT = "mail.daemon.retryonerror.waittime.unit";
```

Comment:
Shouldn't the two new properties have values defined in config.properties and config_properties.html ?